index.js

```diff
@@ -9,5 +9,7 @@ module.exports = {
   sequence: 'sequence',
   chain: 'chain',
   extend: 'extend',
-  extract: 'extract'
+  extract: 'extract',
+  bimap: 'bimap',
```

Comment:
@SimonRichardson Could you release a new version on NPM that would include this changes to `index.js`?